shared/lib/e2e-test-manager.ts

```diff
@@ -58,6 +58,26 @@ export class E2eTestManager {
     return createUser(this.dataSource, additionalData);
   }
 
+  async deleteUserById(id: string) {
+    try {
+      await this.dataSource.getRepository(User).delete({
+        id,
+      });
+    } catch (error) {
+      throw new Error(`Failed to delete test user with id: ${id}`);
+    }
+  }
+
+  async deleteUserByEmail(email: string) {
```

Comment:
Also, it might be easier to just remove a user by passing the user object that you have created, unless you do need to remove then by different properties, but it does not seems you are using that. 

Another option you could do to remove some targeted resource/table entirely, it could be:

```
async deleteTableOf(entity: Entity){
  await this.dataSource.getRepository(entity).delete({})
} 

await testManager.deleteTableOf(User)

```

Reply:
`deleteUserByEmail` is needed `auth.spec` but  I agree in the case of User entity we could just delete the entire table.

Reply:
I missed it! But yeh, when it's needed it's needed. thanks!